ci/unit_tests/functions_test_evaluate/main_data/test_multi_junitxml_ref.eval.junit.xml

```diff
@@ -26,5 +26,17 @@
 		<testcase name="Valid time" time="50"/>
 		<testcase name="Invalid time"/>
 		<testcase/>
+		<testcase name="Error from testing phase - only message">
+			<error message="error message"/>
+		</testcase>
+		<testcase name="Error from testing phase - only type">
+			<error type="error type"/>
+		</testcase>
+		<testcase name="Error from testing phase - message and type">
+			<error message="error message" type="error type"/>
+		</testcase>
+		<testcase name="Error from testing phase - message and type and rest of the test definition">
+			<error message="error message" type="error type"/>
+		</testcase>
```

Comment:
I would expect the diff here ("test message" expected but "test message 2" returned)

Reply:
This is about propagation of error from testing phase, failure (where diff should be visible) could be found on line 23.